book/website/_toc.yml

```
@@ -81,10 +81,14 @@ parts:
       sections:
       - title: Software Licenses
         file: reproducible-research/licensing/licensing-software
-      - title: License Compatibililty
-        file: reproducible-research/licensing/licensing-compatibility
       - title: Data Licenses
         file: reproducible-research/licensing/licensing-data
+      - title: Machine Learning Model Licenses
+        file: reproducible-research/licensing/licensing-ml
+      - title: Case Studies: Choosing an ML License
```

```suggestion
      - title: "Case Studies: Choosing an ML License"
```

The build is failing because of the use of a `:` in the title without being escaped by quotation marks.

YAML is trying to read this as the following.

```
- title.Case Studies: "Choosing an ML License"
```

Which is not valid Jupyter Book config

---

thank you @sgibson91 !

---

woohoo thank you @sgibson91 !